rust_src/src/fns.rs

```diff
@@ -90,6 +94,16 @@ pub fn quote(args: LispCons) -> LispObject {
     args.car()
 }
 
+#[lisp_fn]
+pub fn mapc(function: LispObject, sequence: LispObject) -> LispObject {
+    let leni = length(sequence) as EmacsInt;
+    if unsafe { CHAR_TABLE_P(sequence) } {
```

Comment:
```suggestion
    if sequence.is_char_table() {
```

Reply:
We prefer to use methods where possible instead of porting the C macros.

Reply:
thanks, I missed that one!